### PR TITLE
Refactor commands to use composition

### PR DESCRIPTION
**Pull request by ybogdanov**

### CODE REVIEW

File: src/build/commands.go [hunk 1]
```diff
@@ -72,60 +72,60 @@ type EnvReplacableCommand interface {
...
-		cmd = &CommandExport{cfg}
+		cmd = &CommandExport{CommandBase{cfg}}
 	case "import":
-		cmd = &CommandImport{cfg}
+		cmd = &CommandImport{CommandBase{cfg}}
 	case "arg":
 		cmd = &CommandArg{CommandBase{cfg}}
 	default:
-		return nil, fmt.Errorf("Unknown command: %s", cfg.name)
+		panic(fmt.Sprintf("Unknown command: %s", cfg.name))
```
Comment (kalbasit) [inline src/build/commands.go]: It's generally preferable to not panic especially due to user error and not say `rocker` bug.

File: src/build/commands.go [hunk 2]
```diff
@@ -72,60 +72,60 @@ type EnvReplacableCommand interface {
...
 	}
 
 	if cfg.isOnbuild {
 		cmd = &CommandOnbuildWrap{cmd}
 	}
 
-	return cmd, nil
+	return
```
Comment (kalbasit) [inline src/build/commands.go]: unneeded return.

File: src/build/commands.go (hunk 1 above)
Comment (ybogdanov) [inline src/build/commands.go]: Agree

File: src/build/commands.go (hunk 2 above)
Comment (ybogdanov) [inline src/build/commands.go]:
```bash
$ go build
# github.com/grammarly/rocker/src/build
src/build/commands.go:127: missing return at end of function
$
```

Comment (ybogdanov) [inline src/build/commands.go]:
```bash
$ go version
go version go1.6.3 darwin/amd64
```

Comment (kalbasit) [inline src/build/commands.go]: Oh oops, sorry I did not notice that you are using a named return in the function's signature`func NewCommand(cfg ConfigCommand) (cmd Command) {`. It's required in this case, my apologies.

**Pull request merged**